modules/admin_manual/pages/installation/deployment_recommendations/nfs.adoc

```diff
@@ -0,0 +1,146 @@
+* Use of {nfs-read-write-delegations-url}[Read/Write Delegations].
+
+== Tune the Mount Parameters
+
+=== _netdev
+
+Use this option to ensure that the NFS storage is available before ownCloud starts.
+It is essential when MySQL’s data files are located on the NFS storage, as MySQL could error if the connection to the storage is not ready when ownCloud starts.
+
+TIP: Alternatively, you can use {autofs-url}[autofs] to ensure that mounts are always available.
+
+=== soft
+
+We recommend this option to better handle storage failures.
+If `hard` is used, turning off the ownCloud server is likely to be problematic, and you need a boot and shutdown sequence in the servers.
+
+=== retrans
+
+Use this option to set the number of minor timeouts and retransmissions before major timeouts happen.
+Major timeouts either abort file operations or display the "server not responding" message.
+We recommend that you set it to ?.
+
+=== timeo
+
+Use this option, set in tenths of a second, to set the time before an initial retransmission after a timeout.
+
+According to {nfs-man-page-url}[the NFS man page], better overall performance may be achieved by increasing the timeout when mounting on a busy network, to a slow server, or through several routers or gateways. 
+We recommend that you set it to ?.
+
+=== noasync
+
+In clustered environments, we recommend setting the `noasync` option, to avoid filesystem I/O being handled asynchronously. 
+This has become the default on most distributions, but we recommend checking what this is set to in your ownCloud installations.
+
+=== noac
+
+The `noac` option should be used with caution. 
+Quoting the NFS man page:
+
+[quote, nfs man page]
+Using the noac option provides greater cache coherence among NFS clients accessing the same files, *but it extracts a significant performance penalty*. As such, judicious use of file locking is encouraged instead.
+
+== Tune the Read and Write Block Sizes
+
+The allowed block sizes are the packet chunk sizes that NFS uses when reading and writing data.
+The smaller the size, the greater the number of packets need to be sent to send or receive a file.
+Conversely, the larger the size, the fewer the number of packets need to be sent to send or receive a file.
+
+You can find this information by looking at the `/proc/mounts`, as in the example below.
+
+[source,console]
+----
+cat /proc/mounts | grep -i owncloud
+
+192.168.0.103:/data /mnt/owncloud nfs rw,vers=3,rsize=32768,wsize=32768,hard,proto=tcp,timeo=600,retrans=2,sec=sys,addr=192.168.0.103 0 0
+----
+
+Once you've determined the best sizes, set them by passing the (`rsize` and `wsize`) options when mounting the share or in the share's mount configuration.
+
+.Specifying the read and write block sizes when calling mount
+[source, console]
+----
+mount 192.168.0.104:/data  /mnt -o rsize=65536,wsize=65536
+----
+
+=== Tune the MTU (Maximum Transmission Unit) Size
```

Comment:
Playing with MTU could generate no connection to the other computer, and depends from the devices between them, 1500 is the normal or Jumbo Packages, but IMHO this can be modified if you really know what you are doing. otherwise, you end with a lot of lost packages.

Reply:
I'll add a note to this section.

Reply:
Actually, https://www.networkworld.com/article/2224654/mtu-size-issues.html makes for an excellent discussion on the topic of MTU sizes.

Reply:
First, the connection between server and storage should be non-routed and wired/switched Gigabit.
Second, add a note to check that packages do not get fragmented between server and storage by using the dont fragment option pinging the other side and adopting the mtu value in case of fragments.
Third, any optimisation for a mtu value like 9000 (Jumbo Frames) requires a storage backend being capable saturating gigabit which only high end enterprise storage systems can provide. (technical background on request).

Reply:
additional note regarding jumbo frames: why do you try to fix an issue on the ys level (network) while your storage runs on a ms level (mechanical, if running hdd)... the gain on high end and optimized storage systems is less then 8%

Reply:
I've linked to that article in the changes that I'm pushing shortly.

Reply:
@mmattel, that seems like overkill here. I've linked to the NetworkWorld article which gives an excellent discussion about potential MTU issues.